app/src/androidTest/java/com/github/se/polyfit/ui/screen/IngredientTest.kt

```diff
@@ -331,12 +331,12 @@ class IngredientTest : TestCase() {
   @Ignore("No Potential Ingredients Yet")
   @Test
   fun displayManyPotential() {
```

Comment:
Since you removed the code for potential ingredients, it's best to remove all the tests/references in the test screen as well. The tests to delete should just be all the ignored ones